internal/pkg/bulk/engine.go

```diff
@@ -115,6 +120,22 @@ func (b *Bulker) Client() *elasticsearch.Client {
 	return client
 }
 
+func (b *Bulker) GetRemoteClient(name string) *elasticsearch.Client {
+	remoteEs := b.remoteEsList[name]
+	if remoteEs == nil {
+		return nil
+	}
+	client, ok := remoteEs.(*elasticsearch.Client)
```

Comment:
[Suggestion]
If you need an `*elasticsearch.Client`, use it as the type for the values on `remoteEs`. `SetRemoteClient` doesn't take in an `esapi.Transport`, `Get...` doesn't return one and as `*elasticsearch.Client` implements `esapi.Transport`, it'll be easier if you need to use any `remoteEs` as `esapi.Transport` later.